db/migrations/20211215170939-create-account.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+    up: async(queryInterface, Sequelize) => {
+        await queryInterface.createTable('Accounts', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER
+            },
+            name: {
+                allowNull: false,
+                type: Sequelize.STRING
+            },
+            number: {
+                allowNull: false,
+                type: Sequelize.STRING
+            },
+            balanse: {
```

Comment:
can the balance of an account be null? Maybe it should be equal to 0 by default?

please, add the following code here:
```
allowNull: false,
default: 0,
```